### PR TITLE
Get ksp_plugin_adapter compiling with KSP 1.4.1

### PR DESCRIPTION
**Pull request by aw1621107**

### CODE REVIEW

**Comment by aw1621107** [comment]
I'm stupid and forgot to switch branches before recompiling :(

**Comment by pleroy** [comment]
I am guessing that the extra DLL references will cause our continuous build to protest because we are set up with warning == failure.  We'll see.

**Comment by pleroy** [comment]
Ah, the continuous build is unhappy for a different reason, because the 1.4.1 DLLs don't exist on the build machine.  I'll look into this tomorrow (it's getting late here).

**Comment by pleroy** [comment]
retest this please

File: ksp_plugin_adapter/ksp_plugin_adapter.csproj [hunk 1]
```diff
@@ -57,16 +57,36 @@
...
       <Private>False</Private>
     </Reference>
     <Reference Include="System" />
     <Reference Include="UnityEngine">
-      <HintPath>..\..\KSP Assemblies\1.4.0\UnityEngine.dll</HintPath>
+      <HintPath>..\..\KSP Assemblies\1.4.1\UnityEngine.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
+    <Reference Include="UnityEngine.CoreModule">
+      <HintPath>..\..\KSP Assemblies\1.4.1\UnityEngine.CoreModule.dll</HintPath>
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.csproj]: So as expected these references are generating warnings which cause the continuous build to fail.  There does not seem to be any way to selectively ignore linker warnings in MSBuild.

Ideally, we would want these references to be only active on Mac (and possibly Linux).  According to [this page](http://www.mono-project.com/archived/porting_msbuild_projects_to_xbuild/) it is possible to have references that are conditioned by the operating system.  Could you try this?

Comment (pleroy) [reply ksp_plugin_adapter/ksp_plugin_adapter.csproj]: FYI, on Windows, `$(OS)` is `Windows_NT`.

Comment (aw1621107) [reply ksp_plugin_adapter/ksp_plugin_adapter.csproj]: Hopefully fixed now

**Comment by pleroy** [comment]
Thanks for your contribution.

**Pull request merged**